File: Imp_Test/resources/cs/pizza.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace test
{
    public class Pizza
    {
        public List<string> toppings = new List<string>();

        public void add(string topping)
        {
            if (topping == null)
                return;

            toppings.Add(topping);
        }
    }
}

File: Imp_Test/resources/js/pizza.js
var test = {}

test.Pizza = function() {}
test.Pizza.prototype = {
	toppings: [],
	add: function(topping) {
		var x = 0
		if (topping == null)
			return
		else
			x = 1
		
		this.toppings.push(topping)
	}
}
